### PR TITLE
Sequence load 1b (class attributes for defaults sheet)

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

Added class attributes for defaults sheet metadata and fleshed out and improved some docstrings.

## Affected Issues/Pull Requests

- Partially addresses #824

## Review Notes

See comments in-line.

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [x] [Updated *Unreleased* section of `changelog.md` *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: DataRepo/management/commands/load_table.py [hunk 1]
```diff
@@ -133,12 +141,25 @@ def add_arguments(self, parser):
...
+        parser.add_argument(
+            "--defaults-file",
+            type=str,
+            help=(
+                "Path to a tab-delimited file containing default values.  If --infile is an excel file, you must use "
+                "--defaults-sheet instead.  Required headers: "
+                f"{self.loader_class.get_pretty_default_default_headers()}."
+            ),
+            required=False,
+        )
```
Comment (hepcat72) [inline DataRepo/management/commands/load_table.py:153]: C. Added the ability to supply the defaults in a separate table-like file.

File: DataRepo/management/commands/load_table.py [hunk 2]
```diff
@@ -215,10 +245,136 @@ def handle_wrapper(self, *args, **options):
 
         return handle_wrapper
 
-    def get_sheet(self, default=False):
```
Comment (hepcat72) [inline DataRepo/management/commands/load_table.py:218]: C. In fact, the rest of the changes in this file can be ignored.  2 methods were only moved and the only significant change was a new exception that is raised if options are attempted to be accessed by the derived class before the handle method has been called.

File: DataRepo/utils/loader.py [hunk 3]
```diff
@@ -1062,13 +1087,8 @@ def get_row_val(self, row, header, strip=True):
...
         return val
 
-    def get_user_defaults(self, df, file=None, sheet=None):
-        # TODO: Save these variables in an appropriate place
-        DEFAULT_SHEET = "Defaults" if sheet is None else sheet
-        SHEET_HEADER = "Sheet Name"
-        HEADER_HEADER = "Column Header"
-        DEFAULT_HEADER = "Default Value"
-
+    def get_user_defaults(self):
```
Comment (hepcat72) [inline DataRepo/utils/loader.py:1090]: C. This method started out with method-scope variables.  These were changed to class attributes.

**Review by lparsons — APPROVED** [review]
This PR implements reading a "defaults table" for the loader management commands.  Some changes seem to just move some code around.

**Pull request merged**